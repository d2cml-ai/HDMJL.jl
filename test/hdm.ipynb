{
  "cells": [
    {
      "cell_type": "raw",
      "metadata": {},
      "source": [
        "---\n",
        "title: Plots Demo\n",
        "author: Norah Jones\n",
        "date: 5/22/2021\n",
        "format:\n",
        "  pdf:\n",
        "    code-fold: true\n",
        "---"
      ],
      "id": "d1006825"
    },
    {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
        "## Parametric Plots\n",
        "\n",
        "Plot function pair (x(u), y(u)). \n",
        "See @fig-parametric for an example.\n"
      ],
      "id": "ed194bc4"
    },
    {
      "cell_type": "code",
      "metadata": {},
      "source": [
        "#| label: fig-parametric\n",
        "#| fig-cap: Parametric Plots\n",
        "\n",
        "12 + 12\n",
        "\n",
        "llk() = print(\"sakjfalsj sa\")\n",
        "\n",
        "llk()"
      ],
      "id": "fig-parametric",
      "execution_count": null,
      "outputs": []
    }
  ],
  "metadata": {
    "kernelspec": {
      "name": "julia-1.7",
      "language": "julia",
      "display_name": "Julia 1.7.3"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 5
}