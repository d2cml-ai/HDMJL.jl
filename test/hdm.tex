\documentclass[12pt,a4paper]{article}

\usepackage[a4paper,text={16.5cm,25.2cm},centering]{geometry}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{bm}
\usepackage{graphicx}
\usepackage{microtype}
\usepackage{hyperref}
\setlength{\parindent}{0pt}
\setlength{\parskip}{1.2ex}

\hypersetup
       {   pdfauthor = { Norah Jones },
           pdftitle={ Plots Demo },
           colorlinks=TRUE,
           linkcolor=black,
           citecolor=blue,
           urlcolor=blue
       }

\title{ Plots Demo }

\author{ Norah Jones }

\date{ 5/22/2021 }

\usepackage{upquote}
\usepackage{listings}
\usepackage{xcolor}
\lstset{
    basicstyle=\ttfamily\footnotesize,
    upquote=true,
    breaklines=true,
    breakindent=0pt,
    keepspaces=true,
    showspaces=false,
    columns=fullflexible,
    showtabs=false,
    showstringspaces=false,
    escapeinside={(*@}{@*)},
    extendedchars=true,
}
\newcommand{\HLJLt}[1]{#1}
\newcommand{\HLJLw}[1]{#1}
\newcommand{\HLJLe}[1]{#1}
\newcommand{\HLJLeB}[1]{\textcolor[RGB]{150,0,80}{\colorbox[RGB]{30,0,16}{#1}}}
\newcommand{\HLJLo}[1]{#1}
\newcommand{\HLJLk}[1]{\textcolor[RGB]{24,172,200}{#1}}
\newcommand{\HLJLkc}[1]{#1}
\newcommand{\HLJLkd}[1]{#1}
\newcommand{\HLJLkn}[1]{\textcolor[RGB]{249,38,114}{#1}}
\newcommand{\HLJLkp}[1]{#1}
\newcommand{\HLJLkr}[1]{#1}
\newcommand{\HLJLkt}[1]{#1}
\newcommand{\HLJLn}[1]{#1}
\newcommand{\HLJLna}[1]{\textcolor[RGB]{117,171,3}{#1}}
\newcommand{\HLJLnb}[1]{#1}
\newcommand{\HLJLnbp}[1]{#1}
\newcommand{\HLJLnc}[1]{\textcolor[RGB]{117,171,3}{#1}}
\newcommand{\HLJLncB}[1]{\textcolor[RGB]{24,172,200}{#1}}
\newcommand{\HLJLnd}[1]{\textcolor[RGB]{117,171,3}{#1}}
\newcommand{\HLJLne}[1]{#1}
\newcommand{\HLJLneB}[1]{\textcolor[RGB]{117,171,3}{#1}}
\newcommand{\HLJLnf}[1]{\textcolor[RGB]{117,171,3}{#1}}
\newcommand{\HLJLnfm}[1]{\textcolor[RGB]{117,171,3}{#1}}
\newcommand{\HLJLnp}[1]{#1}
\newcommand{\HLJLnl}[1]{#1}
\newcommand{\HLJLnn}[1]{#1}
\newcommand{\HLJLno}[1]{\textcolor[RGB]{117,171,3}{#1}}
\newcommand{\HLJLnt}[1]{\textcolor[RGB]{249,38,114}{#1}}
\newcommand{\HLJLnv}[1]{#1}
\newcommand{\HLJLnvc}[1]{#1}
\newcommand{\HLJLnvg}[1]{#1}
\newcommand{\HLJLnvi}[1]{#1}
\newcommand{\HLJLnvm}[1]{#1}
\newcommand{\HLJLl}[1]{\textcolor[RGB]{108,76,214}{#1}}
\newcommand{\HLJLld}[1]{\textcolor[RGB]{170,58,168}{#1}}
\newcommand{\HLJLs}[1]{\textcolor[RGB]{170,58,168}{#1}}
\newcommand{\HLJLsa}[1]{\textcolor[RGB]{170,58,168}{#1}}
\newcommand{\HLJLsb}[1]{#1}
\newcommand{\HLJLsc}[1]{#1}
\newcommand{\HLJLsd}[1]{\textcolor[RGB]{170,58,168}{#1}}
\newcommand{\HLJLsdB}[1]{#1}
\newcommand{\HLJLsdC}[1]{#1}
\newcommand{\HLJLse}[1]{\textcolor[RGB]{108,76,214}{#1}}
\newcommand{\HLJLsh}[1]{#1}
\newcommand{\HLJLsi}[1]{#1}
\newcommand{\HLJLso}[1]{#1}
\newcommand{\HLJLsr}[1]{#1}
\newcommand{\HLJLss}[1]{#1}
\newcommand{\HLJLssB}[1]{#1}
\newcommand{\HLJLnB}[1]{\textcolor[RGB]{108,76,214}{#1}}
\newcommand{\HLJLnbB}[1]{\textcolor[RGB]{108,76,214}{#1}}
\newcommand{\HLJLnfB}[1]{#1}
\newcommand{\HLJLnh}[1]{#1}
\newcommand{\HLJLni}[1]{#1}
\newcommand{\HLJLnil}[1]{#1}
\newcommand{\HLJLnoB}[1]{#1}
\newcommand{\HLJLoB}[1]{\textcolor[RGB]{249,38,114}{#1}}
\newcommand{\HLJLow}[1]{#1}
\newcommand{\HLJLp}[1]{#1}
\newcommand{\HLJLc}[1]{\textcolor[RGB]{136,136,136}{#1}}
\newcommand{\HLJLch}[1]{\textcolor[RGB]{136,136,136}{#1}}
\newcommand{\HLJLcm}[1]{#1}
\newcommand{\HLJLcp}[1]{#1}
\newcommand{\HLJLcpB}[1]{\textcolor[RGB]{136,136,136}{#1}}
\newcommand{\HLJLcs}[1]{#1}
\newcommand{\HLJLcsB}[1]{#1}
\newcommand{\HLJLg}[1]{#1}
\newcommand{\HLJLgd}[1]{\textcolor[RGB]{249,38,114}{#1}}
\newcommand{\HLJLge}[1]{\textit{#1}}
\newcommand{\HLJLgeB}[1]{#1}
\newcommand{\HLJLgh}[1]{#1}
\newcommand{\HLJLgi}[1]{\textcolor[RGB]{117,171,3}{#1}}
\newcommand{\HLJLgo}[1]{#1}
\newcommand{\HLJLgp}[1]{#1}
\newcommand{\HLJLgs}[1]{\textbf{#1}}
\newcommand{\HLJLgsB}[1]{\textcolor[RGB]{136,136,136}{#1}}
\newcommand{\HLJLgt}[1]{#1}


\begin{document}

\maketitle

\section{Parametric Plots}
Plot function pair (x(u), y(u)).  See @fig-parametric for an example.


\begin{lstlisting}
(*@\HLJLni{12}@*) (*@\HLJLoB{+}@*) (*@\HLJLni{12}@*)

(*@\HLJLnf{llk}@*)(*@\HLJLp{()}@*) (*@\HLJLoB{=}@*) (*@\HLJLnf{print}@*)(*@\HLJLp{(}@*)(*@\HLJLs{"{}sakjfalsj}@*) (*@\HLJLs{sa"{}}@*)(*@\HLJLp{)}@*)

(*@\HLJLnf{llk}@*)(*@\HLJLp{()}@*)
\end{lstlisting}

\begin{lstlisting}
sakjfalsj sa
\end{lstlisting}


\section{Adicional}

\begin{lstlisting}
(*@\HLJLk{using}@*) (*@\HLJLn{HDMjl}@*)
(*@\HLJLn{rlasso}@*)
\end{lstlisting}

\begin{lstlisting}
rlasso (generic function with 1 method)
\end{lstlisting}


\begin{lstlisting}
(*@\HLJLcs{{\#}{\#}{\#}{\#}{\#}}@*)
(*@\HLJLk{using}@*) (*@\HLJLn{Plots}@*)

(*@\HLJLnf{plot}@*)(*@\HLJLp{(}@*)(*@\HLJLn{sin}@*)(*@\HLJLp{,}@*) 
     (*@\HLJLn{x}@*)(*@\HLJLoB{->}@*)(*@\HLJLnf{sin}@*)(*@\HLJLp{(}@*)(*@\HLJLni{2}@*)(*@\HLJLn{x}@*)(*@\HLJLp{),}@*) 
     (*@\HLJLni{0}@*)(*@\HLJLp{,}@*) 
     (*@\HLJLni{2}@*)(*@\HLJLn{\ensuremath{\pi}}@*)(*@\HLJLp{,}@*) 
     (*@\HLJLn{leg}@*)(*@\HLJLoB{=}@*)(*@\HLJLkc{false}@*)(*@\HLJLp{,}@*) 
     (*@\HLJLn{fill}@*)(*@\HLJLoB{=}@*)(*@\HLJLp{(}@*)(*@\HLJLni{0}@*)(*@\HLJLp{,}@*)(*@\HLJLoB{:}@*)(*@\HLJLn{lavender}@*)(*@\HLJLp{))}@*)
\end{lstlisting}

\includegraphics[width=\linewidth]{jl_SdKYyO/hdm_3_1.pdf}


\end{document}
