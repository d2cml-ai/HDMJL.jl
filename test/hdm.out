\BOOKMARK [1][-]{section.1}{\376\377\000P\000a\000r\000a\000m\000e\000t\000r\000i\000c\000\040\000P\000l\000o\000t\000s}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000A\000d\000i\000c\000i\000o\000n\000a\000l}{}% 2
